command_cursor.c

```diff
@@ -323,10 +323,6 @@ PHP_METHOD(MongoCommandCursor, valid)
 
 	MONGO_CHECK_INITIALIZED(cmd_cursor->zmongoclient, MongoCommandCursor);
 
-	if (!cmd_cursor->started_iterating) {
-		RETURN_FALSE;
-	}
-
```

Comment:
Why did you remove this? I realize that the case might never be hit, but then again, you never know. Leaving it in is IMO safer. Add a comment instead?